### DISCUSSION TITLE
How to stop docker container/program when download has completed?

### DISCUSSION THREAD

**Discussion by PylotLight** [mxpv/podsync#559] [General]
I've setup podsync in a GitHub actions pipe to run on a schedule to download files and push to an s3 bucket. One of the issues I ran into is that it's designed to run like a continuous server rather than a triggered single-use application to be run on a schedule. Is there any flags or controls that can be used to end the server once download/upload is complete?

p.s I do have a workaround by timing out the step and adding continue on error, but this is not ideal and just thought I'd ask.

**Comment by Harnas** [comment]
Just run with `--headless` parameter.

**Reply by PylotLight** [reply]
Yep that worked.
